src/audio.h

```diff
@@ -79,3 +79,5 @@ void fade_bgm(double fadetime);
 void resume_bgm(void);
 void save_bgm(void); // XXX: this is broken
 void restore_bgm(void); // XXX: this is broken
+
+void mute_audio_handler(void);
```

Comment:
Internal functions should be declared static, and not exposed in the headers. I'm aware that not all code follows this rule, but that just means it should be fixed/refactored.